geopyspark/tests/io_tests/s3_geotiff_rdd_test.py

```diff
@@ -67,26 +68,36 @@ def read_multiband_geotrellis(self, opt=options):
                      s3_client=opt['s3Client'],
                      max_tile_size=opt.get('maxTileSize'))
 
-        return [tile[1] for tile in result.to_numpy_rdd().collect()]
+        return result
 
-    def test_whole_tiles(self):
-        geotrellis_tiles = self.read_multiband_geotrellis()
+    def test_segment_tiles(self):
+        # GeoTrellis will read GeoTiff Segments given no window size
+        # Retile them to match Rasterio read and check the cell values
+        geotrellis_tiles = self.read_multiband_geotrellis().tile_to_layout(LocalLayout(512))
+        geotrellis_tiles = [tile[1] for tile in geotrellis_tiles.to_numpy_rdd().collect()]
         rasterio_tiles = self.read_geotiff_rasterio([self.file_path], False)
 
         for x, y in zip(geotrellis_tiles, rasterio_tiles):
-            self.assertTrue((x.cells == y['cells']).all())
+            print('\n')
```

Comment:
They were really helpful when things went wrong and are otherwise hidden, I'd actually like to save them for when/if this throws again.